server/utils.py

```diff
@@ -87,3 +87,16 @@ def validate_uuid4(uuid_string):
     # valid uuid4. This is bad for validation purposes.
 
     return str(val) == uuid_string
+
+
+def is_ip_allowed(request, shops_IP_whitelist, shop_id):
+    if (shop_id in shops_IP_whitelist):
```

Comment:
more pythonesk to use: shops_IP_whitelist.get(shop_id)

Reply:
i ll do that in the next commits